### PR TITLE
Video

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
Added Single Video page
-added single video and recommedned videos
-added add to playlist modal

snap shot:
image

preview link:
https://deploy-preview-3--dreamy-stroopwafel-3d0999.netlify.app/

### CODE REVIEW

**Review by Muzammil-khan710 — APPROVED** [review]
Ui is good.
good folder structure
good to merge

File: src/components/playlistModal/playlistModal.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    overflow-y:scroll;
+    align-items: flex-start;
+    padding-left:0.7rem; 
+    gap:5px;
+  }
+  
+  .create-playlist{
+    text-align:center;
+    padding:10px;
+    padding-bottom:10px;
```
Comment (Adarsh-Patel-Dev) [inline src/components/playlistModal/playlistModal.css:39]: you can use rem here

File: src/components/videoListing/videoListing.js [hunk 2]
```diff
@@ -1,55 +1,48 @@
...
-    )
+  const { apiVideos } = useApi();
+  const { setSingleVideo } = useSingleVideo();
+
+  return (
+    <div className="videosListing">
+      {apiVideos.map((item) => {
+        return (
+          <>
+            {" "}
```
Comment (Adarsh-Patel-Dev) [inline src/components/videoListing/videoListing.js]: remove this

File: src/components/videoListing/videoListing.js [hunk 3]
```diff
@@ -1,55 +1,48 @@
...
+                  className="img-thumbnail"
+                  src={item.thumbnailUrl}
+                  alt={item.title}
+                />
+                <h1 className="play">
+                  <i className="fas fa-play"></i>play
+                </h1>
+                </Link>
+                <div className="video-description">
+                  <img className="video-logo" src={item.logoUrl} alt="log" />
```
Comment (Adarsh-Patel-Dev) [inline src/components/videoListing/videoListing.js]: alt attribute can be "logo"

File: src/context/apiContext/api.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React,{ useContext,createContext,useEffect,useState } from 'react'
+import axios from "axios"
+
+const apiContext=createContext(null);
+
+
```
Comment (Adarsh-Patel-Dev) [inline src/context/apiContext/api.js]: remove extra line breaks

**Pull request merged**